ubuntu/Dockerfile

```diff
@@ -29,14 +29,14 @@ RUN set -ex; \
 	&& chown -R kong:0 /usr/local/kong \
 	&& chown kong:0 /usr/local/bin/kong \
 	&& chmod -R g=u /usr/local/kong \
-	&& kong version
+    && if [ "$ASSET" = "ce" ] ; then \
+	kong version ; \
+    fi;
```

Comment:
spacing looks slightly inconsistent

Reply:
Used spaces, I think now it's consistent with the `if` on top. Is that what you meant, or you meant regarding the other Dockerfiles?